### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by aleks-ivanov** [Keriew/augustus#434] [General]
Hello person(s) behind this mod 🙂 

Sincerely, thank you! 

Immense respect and appreciation for your effort!

**Comment by OldLimey** [comment]
Seconded! Big thank you to all the team involved. I was recently disabled in an accident and had to give up my first love of racing simulations like rFactor2 because of nerve damage limiting my coordination and dexterity. I have returned to good old fashioned city building games and Augustus has breathed new life into Caesar3, a favorite of mine from decades past, and made it really special!
